mllib-dal/src/main/scala/com/intel/oap/mllib/clustering/KMeansDALImpl.scala

```diff
@@ -27,6 +27,18 @@ import org.apache.spark.mllib.clustering.{KMeansModel => MLlibKMeansModel}
 import org.apache.spark.mllib.linalg.{Vector => OldVector, Vectors => OldVectors}
 import org.apache.spark.rdd.RDD
 
+
+class KMeansTimerClass() extends Utils.AlgoTimeMetrics{
```

Comment:
Can we move all TimerClass into single file without scattering into different Impl files ?